exercise1/exercise9/Main.cpp

```diff
@@ -0,0 +1,40 @@
+/*Exercise 9 - Shopping
+Write a program that will simulate a shop where you can buy different items. Let the user choose how many items of each type they want, and then output the final price. Make sure to combine integer and floating-point types
+*/
+#include <iostream>
+using namespace std;
+
+int main() 
+{
+	int item1, item2, item3, price1, price2, price3, op1, op2, op3, suma;
```

Comment:
Same here, you don't need to declare your variables before everything else. There is no gain and in fact, the code is harder to read and follow.

Reply:
It seems like you're not taking into account all feedback. Please fix all comments before requesting a new review.